README.md

```diff
@@ -1,34 +1,97 @@
+[![Medium](https://img.shields.io/badge/Medium-gray?logo=medium)](https://medium.com/astar-network)
 
-## Trying it out
+</div>
 
-Simply go to [docs.astar.network](https://docs.astar.network) and follow the 
-[installation](https://docs.astar.network/build/smart-contracts) instructions. You can 
-also try out one of the [core features](https://docs.astar.network/learn/core-features).
+Astar Network is an interoperable blockchain based the Substrate framework and the hub for dApps within the Polkadot Ecosystem.
+With Astar Network and Shiden Network, people can stake their tokens to a Smart Contract for rewarding projects that provide value to the network.
 
-## Contributing
+For contributing to this project, please read our [Contribution Guideline](./CONTRIBUTING.md).
 
-Currently, we have 2 production networks:
+## Building From Source
 
-- [Astar Network](https://astar.network/): formerly known as Plasm Network, and our future Parachain on Polkadot Network
-- [Shiden Network](https://shiden.astar.network/): Parachain of Kusama Network and our canary release for Astar Network
+> This section assumes that the developer is running on either macOS or Debian-variant operating system. For Windows, although there are ways to run it, we recommend using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) or from a virtual machine for stability.
 
-And 2 test networks:
+Execute the following command from your terminal to set up the development environment and build the node runtime.
 
-- [Dusty Network](https://github.com/PlasmNetwork/Astar/tree/development/dusty): an independent test network with the latest cutting edge features before it is introduced to Astar Network
-- [Shibuya Network](https://github.com/PlasmNetwork/Astar/tree/development/shibuya): a Parachain test network with the latest cutting edge features related to XMCP, collators, and other cross chain operations that is planned to be rolled out to Shiden Network
+```bash
+# install Substrate development environment via the automatic script
+$ curl https://getsubstrate.io -sSf | bash -s -- --fast
 
-In the future we will gradually change this structure to have a single test network.
-For details, please read the [Contribution Guidelines](./CONTRIBUTING.md).
+# clone the Git repository
+$ git clone --recurse-submodules https://github.com/PlasmNetwork/Astar.git
 
-If you would like to be the validator, please check out [our tutorial](https://docs.astar.network/build/validator-guide) and join [Discord tech channel](https://discord.gg/wUcQt3R).
+# change current working directory
+$ cd Astar
 
-## LICENSE
+# compile the node
+# note: you may encounter some errors if `wasm32-unknown-unknown` is not installed, or if the toolchain channel is outdated
+$ cargo build --release
 
-Astar is licensed under the GPLv3.0 by Stake Technologies Inc.
+# run a temporary developer node locally
+# `astar-collator --help` for more commands
+$ ./target/release/astar-collator --dev --tmp
```

Comment:
Not sure that it will work now, in future it could work

Reply:
For now, that’s about to join testnet?

Reply:
I'll write `./astar-collator --help` to make it ambiguous until we can allow the users to setup a local dev node